specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -73,29 +73,72 @@ Legend (terms and table are based on Pragmatic Marketing recommendations):
+| Error prone                                                                                               | 1 <br /> this solution might have many more cells → they can be omitted or added with no intention                                   | 2 <br /> will have less cells and the prints will not be in the run() → it will be less likely to add something that is not intended to be in run() |
+| File structure (the user decides what the sections will be and where they will be located)                | 2                                                                                                                                    | 2                                                                                                                                                   |
+| Resembles the current run function                                                                        | 1                                                                                                                                    | 2                                                                                                                                                   |
+| Easy for testing(direct prints)                                                                           | 3 <br /> requires just a new cell to be created and printed; which can be between the magic commands; they will not have the vdk tag | 2 <br /> requires the functions to be called in a different section → not in run(); still does not require that much effort                         |
+| Usability: is the site easy to use by core users                                                          | 3 <br /> it really resembles the standard way of working with Jupyter                                                                | 1 <br /> it requires more programming knowledge to separate the run() into many other functions and to use the run as a caller                      |
+| Error preventions                                                                                         | 1                                                                                                                                    | 1                                                                                                                                                   |
+| Ease of learning: how fast a user who has never seen the user interface before can accomplish basic tasks | 1 <br /> it is harder since the magic commands and writing a function to multiple cells is Jupyter specific                          | 2 <br />  calling functions from one function is a thing we do generally in coding                                                                  |
+
+    Solution 2 total score: 18       Solution 1 total score: 23
+Final decision: Solution 1
```

Comment:
Mention that both solutions are not Mutually Exclusive. That is at some point we can implement both. But for the first iteration/version based on above analysis we will go with Solution 1.